src/thunderbots/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ai/hl/stp/play/kickoff_enemy_play.h"
+
+#include "ai/hl/stp/evaluation/enemy_threat.h"
+#include "ai/hl/stp/evaluation/possession.h"
+#include "ai/hl/stp/play/play_factory.h"
+#include "ai/hl/stp/tactic/move_tactic.h"
+#include "ai/hl/stp/tactic/shadow_kickoff_tactic.h"
+#include "shared/constants.h"
+
+const std::string KickoffEnemyPlay::name = "KickoffEnemy Play";
+
+std::string KickoffEnemyPlay::getName() const
+{
+    return KickoffEnemyPlay::name;
+}
+
+bool KickoffEnemyPlay::isApplicable(const World &world) const
+{
+    return world.gameState().isTheirKickoff();
+}
+
+bool KickoffEnemyPlay::invariantHolds(const World &world) const
+{
+    return world.gameState().isTheirKickoff();
+}
+
+void KickoffEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield)
+{
+    // TODO: This needs to be a goalie tactic
+    auto lone_goalie_tactic_0 = std::make_shared<MoveTactic>(true);
+
+    // these tactics will be assigned
+    auto shadow_kickoff_tactic_1 =
+        std::make_shared<ShadowKickoffTactic>(world.field(), true);
+    auto shadow_kickoff_tactic_2 =
+        std::make_shared<ShadowKickoffTactic>(world.field(), true);
+    auto shadow_kickoff_tactic_3 =
+        std::make_shared<ShadowKickoffTactic>(world.field(), true);
+
+    // move tactic for robots to defend net
+    auto move_tactic_4 = std::make_shared<MoveTactic>(true);
+    auto move_tactic_5 = std::make_shared<MoveTactic>(true);
+
+    do
+    {
+        // TODO: Replace placeholder tactic with goalie tactic
+        lone_goalie_tactic_0->updateParams(Point(4, 0), Angle::half(), 0);
```

Comment:
You'll want to make this move to `world.field().friendlyGoal()` so it adjusts based on what side we're defending for now.

Reply:
fixed